UI/MapWnd.h

```diff
@@ -65,19 +65,22 @@ class MapWnd : public GG::Wnd {
       * save-and-load cycle */
     void                        GetSaveGameUIData(SaveGameUIData& data) const;
 
-    /** returns true iff this MapWnd is visible and usable for interaction, but
-      * the allowed interactions are restricted to those appropriate to the
-      * production screen */
+    /** returns true if MapWnd is visible and usable behind a production window.
+     * MapWnd interactions are restricted to those appropriate to the production window */
     bool                        InProductionViewMode() const;
 
-    /** returns true iff this MapWnd is visible and usable for interaction, but
-      * the allowed interactions are restricted to those appropriate to the
-      * research screen */
+    /** returns true if MapWnd is visible and usable behind a research window.
+     * MapWnd interactions are restricted to those appropriate to the research window.
```

Comment:
The original comment was on InProductionViewMode from 2005-09-01.

The use of InProductionViewMode is to disable Fleet::MouseHere() when in the production window with the map window visible.

The use of the two new functions is to disable map hotkeys when in the research/production windows.

The second part of the comment is to indicates expected usage of the functions to create 4 modal states for the user interface: map only, production, research and design.  In each user interface state the map window may have a different subset of available actions.